### ISSUE TITLE
Throttling and doubt

### ISSUE THREAD

**Issue by Elessar1802** [rodmaureirac/thinkpad-e14-linux#5]
I have found using erPalma's throttled improves performance. Also please pardon me i don't know the guidelines to be followed on this platform. Also on gnome, there is frame rate drop or jittering on the activities overview animation if more than 1 window is open, is there any solution for this?

**Comment by rodmaureirac** [comment]
Thanks for throttling info. I'll test erPalma's patch this weekend.
About GNOME, at least for me, it seems to work fine (Fedora 33). But also keep in mind animations on GNOME are quite laggy.
- [GNOME 3.38 animation thread](https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3428)

**Comment by Elessar1802** [comment]
Using throttled gives me a 500 points boost in geekbench 5. 3301 multi score performance. I am currently playing with tlp. I found using energy_performance on performance, keeping scaling governor at powersave,  improves the gnome experience. Using it at balance_power makes the animations very sluggish. Power draw sits at around 8000mW using tlp-stat -b.

**Comment by Elessar1802** [comment]
I saw the new README under intel throttling. I would like to point out that erpalma's fix applies those power limits as a way to overcome the throttling issue. I don't why that throttling issue occurs, but those values mentioned there actually is used to overcome that throttling issue. This throttling issue has been there for many lenovo laptops aswell. Also, the throttling has nothing to do with animations actually. I had looked up about throttling because I was getting low geekbench5 score. Thanks for listening to my query and adding it to this repo.
